presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPushAggregationThroughOuterJoin.java

```diff
@@ -67,14 +53,9 @@ public void testPushesAggregationThroughLeftJoin()
                                         Optional.empty(),
                                         Optional.empty(),
                                         Optional.empty()
-                                ),
-                                ImmutableMap.of(p.symbol("AVG", DOUBLE),
-                                        new AggregationNode.Aggregation(averageFunctionCall, avgSignature, Optional.empty())),
-                                ImmutableList.of(ImmutableList.of(p.symbol("COL1", BIGINT))),
-                                SINGLE,
-                                Optional.empty(),
-                                Optional.empty())
-                )
+                                ))
+                        .addAggregation(p.symbol("AVG", DOUBLE), PlanBuilder.expression("avg(COL2)"), ImmutableList.of(DOUBLE))
+                        .groupingSets(ImmutableList.of(ImmutableList.of(p.symbol("COL1", BIGINT))))))
```

Comment:
Looks awful. Maybe we could have `.groupBy(Symbol... )` for cases like this?
That would remove need for `ImmutableList.of(ImmutableList.of(...))`.